riscv-test-suite/rv32i/Makefrag

```diff
@@ -80,8 +77,6 @@ rv32i_sc_tests = \
     I-CSRRSI-01 \
     I-CSRRC-01 \
     I-CSRRCI-01 \
-    I-ECALL-01 \
-    I-EBREAK-01 \

```

Comment:
Why are these removed?  Are they just tests that OpenTitan currently fails?

Reply:
I should have clarified. These are tests that currently fail on ibex. 
so they are also removed for OpenTitan.  I'll add a note in the readme.

Reply:
Can you move them to the bottom and comment them out?  Just to make it easier to re-enable them in the future?

Reply:
moved to the bottom. please take a took.  Also added to README.